dipy/data/fetcher.py

```diff
@@ -1628,8 +1635,141 @@ def read_five_af_bundles():
 
         bundles = []
         for sub in subjects:
-            fname = os.path.join(temp_dir, sub, 'AF_L.trk')
+            fname = pjoin(temp_dir, sub, 'AF_L.trk')
             bundle_obj = load_trk(fname, 'same', bbox_valid_check=False)
             bundles.append(bundle_obj.streamlines)
 
     return bundles
+
+
+def to_bids_description(path, fname='dataset_description.json',
+                        BIDSVersion="1.4.0", **kwargs):
+    """Dumps a dict into a bids description at the given location"""
+    kwargs.update({"BIDSVersion": BIDSVersion})
+    desc_file = op.join(path, fname)
+    with open(desc_file, 'w') as outfile:
+        json.dump(kwargs, outfile)
+
+
+def fetch_hcp(subjects,
+              hcp_bucket='hcp-openaccess',
+              profile_name="hcp",
+              path=None,
+              study='HCP_1200',
+              aws_access_key_id=None,
+              aws_secret_access_key=None):
+    """
+    Fetch HCP diffusion data and arrange it in a manner that resembles the
+    BIDS [1]_ specification.
+
+    Parameters
+    ----------
+    subjects : list
+        Each item is an integer, identifying one of the HCP subjects
+    hcp_bucket : string, optional
+        The name of the HCP S3 bucket. Default: "hcp-openaccess"
+    profile_name : string, optional
+        The name of the AWS profile used for access. Default: "hcp"
+    path : string, optional
+        Path to save files into. Default: '~/AFQ_data'
+    study : string, optional
+        Which HCP study to grab. Default: 'HCP_1200'
+    aws_access_key_id : string, optional
+        AWS credentials to HCP AWS S3. Will only be used if `profile_name` is
+        set to False.
+    aws_secret_access_key : string, optional
+        AWS credentials to HCP AWS S3. Will only be used if `profile_name` is
+        set to False.
+
+    Returns
+    -------
+    dict with remote and local names of these files,
+    path to BIDS derivative dataset
+
+    Notes
+    -----
+    To use this function with its default setting, you need to have a
+    file '~/.aws/credentials', that includes a section:
```

Comment:
What will be the path on Windows?

Reply:
This is what AWS has to say about that: https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html#cli-configure-files-where, but I am not sure that I understand it. What's the deal with `%UserProfile%`? Maybe I should just include a link to this webpage?

Reply:
yes, can you add this link ?